app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -69,7 +70,6 @@
     StorageReference storageReference;
     String userId;
     String organizerId;
-    ArrayList<String> participantNames = new ArrayList<>();
     StringBuilder participantNamesString = new StringBuilder();
 
     ImageView activityImage;
```

Comment:
On lines 101 and 102 further down, you're requesting backend instances from `FirebaseStorage` and `FirebaseFirestore` directly -> please use the `BackendInstanceProvider` class! It'll make for easier Firebase-less testing

(Sorry, I couldn't comment on those lines directly)

Reply:
Yes this is something that has to be done, effectively. I guess this is not the only class in which we will have to replace our `.getInstance()` to Firebase services. I know it's a routine replacement to do , but did you already use your `BackendInstanceProvider` class somewhere in our project so that I can have an example of how to use it?

If I remember correctly, your `BackendInstanceProvider` usage would look as follows:
`FirebaseFirestore fStore = BackendInstanceProvider.getFirestoreInstance()`

Is it correct?

Reply:
Yup, exactly!